src/components/Chips/Chips.tsx

```diff
@@ -183,72 +190,81 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
           style: backgroundColorStyle
         }
       : {
+          tabIndex: isOverflowing ? 0 : undefined,
+          "aria-label": isOverflowing ? ariaLabel || label : undefined,
```

Comment:
did not get this change, if it's not overflowing we want not aria label? What the aria label is affected by overflow and not always ariaLabel || label

Reply:
Fixed